gazebo_ros/CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ ament_export_dependencies(gazebo_dev)
 
 if(BUILD_TESTING)
   add_subdirectory(test)
+  find_package(ament_lint_auto REQUIRED)
+  ament_lint_auto_find_test_dependencies()
```

Comment:
Thanks for catching this! I was really suspicious that suddenly I was not making any new linting errors :smile: On my machine, I need these extra changes for all tests to pass, is the same happening to you?

```diff
diff --git a/gazebo_ros/launch/empty_world.launch.py b/gazebo_ros/launch/empty_world.launch.py
index 6d42d90..aae73c1 100644
--- a/gazebo_ros/launch/empty_world.launch.py
+++ b/gazebo_ros/launch/empty_world.launch.py
@@ -15,7 +15,7 @@
 """Launch a Gazebo server with an empty world and initialize ROS with command line arguments."""
 
 import launch
-import launch_ros.actions
+
 
 def generate_launch_description():
     # TODO(anyone): Forward command line arguments once that's supported, see
@@ -28,4 +28,3 @@ def generate_launch_description():
     return launch.LaunchDescription([
         gzserver
     ])
-
```

Reply:
Ah my flake8 was broken so it wasn't showing this... patch applied